proof/ec_opts.saw

```diff
@@ -204,13 +211,14 @@ POINTonE2_add_ov <- verify "POINTonE2_add" assembly_overrides POINTonE2_add_spec
 POINTonE2_add_alias_ov <- verify "POINTonE2_add" assembly_overrides POINTonE2_add_alias_spec;
 POINTonE1_add_affine_ov <- verify "POINTonE1_add_affine" assembly_overrides POINTonE1_add_affine_spec;
 POINTonE1_add_affine_alias_ov <- verify "POINTonE1_add_affine" assembly_overrides POINTonE1_add_affine_alias_spec;
-POINTonE2_add_affine_ov <- crucible_llvm_verify m "POINTonE2_add_affine" assembly_overrides false POINTonE2_add_affine_spec abc;
+POINTonE2_add_affine_ov <- verify "POINTonE2_add_affine" assembly_overrides POINTonE2_add_affine_spec;
 POINTonE2_add_affine_alias_ov <- verify "POINTonE2_add_affine" assembly_overrides POINTonE2_add_affine_alias_spec;
 POINTonE1_double_ov <- verify "POINTonE1_double" assembly_overrides POINTonE1_double_spec;
 POINTonE1_double_alias_ov <- verify "POINTonE1_double" assembly_overrides POINTonE1_double_alias_spec;
 POINTonE2_double_ov <- verify "POINTonE2_double" assembly_overrides POINTonE2_double_spec;
 POINTonE2_double_alias_ov <- verify "POINTonE2_double" assembly_overrides POINTonE2_double_alias_spec;
 POINTonE1_is_equal_ov <- verify "POINTonE1_is_equal" assembly_overrides POINTonE1_is_equal_spec;
 POINTonE2_is_equal_ov <- verify "POINTonE2_is_equal" assembly_overrides POINTonE2_is_equal_spec;
+//POINTonE1_mult_w5_ov <- really_verify "POINTonE1_mult_w5" assembly_overrides (POINTonE1_mult_w5_spec 636); // TODO: how many bits?
```

Comment:
is this used?

Reply:
fixed: it's now verified for the right number of bits